src/components/NftRewards/NftTierCard.tsx

```diff
@@ -193,11 +193,8 @@ export function NftTierCard({
                 title={false}
                 paragraph={{ rows: 1, width: ['50%'] }}
               >
-                <span
-                  className="mt-2 text-grey-500"
-                  style={{ fontSize: '0.6rem' }}
```

Comment:
cc @johnnyd-eth should always use tailwind font-sizes going forward. shouldn't need a font size lower than text-xs (0.75rem) ever - if so then the design needs revisiting prob